openshift/templates/db-backup/mongo/db-backup.dc.json

```diff
@@ -0,0 +1,590 @@
+                      "secretKeyRef": {
+                        "name": "${FTP_SECRET_KEY}",
+                        "key": "ftp-user"
+                      }
+                    }
+                  },
+                  {
+                    "name": "FTP_PASSWORD",
+                    "valueFrom": {
+                      "secretKeyRef": {
+                        "name": "${FTP_SECRET_KEY}",
+                        "key": "ftp-password"
+                      }
+                    }
+                  },
+                  {
+                    "name": "WEBHOOK_URL",
+                    "valueFrom": {
+                      "secretKeyRef": {
+                        "name": "${NAME}",
+                        "key": "webhook-url"
+                      }
+                    }
+                  },
+                  {
+                    "name": "ENVIRONMENT_FRIENDLY_NAME",
+                    "value": "${ENVIRONMENT_FRIENDLY_NAME}"
+                  },
+                  {
+                    "name": "ENVIRONMENT_NAME",
+                    "value": "${ENVIRONMENT_NAME}"
+                  }
+                ],
+                "resources": {
+                  "requests": {
+                    "cpu": "${CPU_REQUEST}",
+                    "memory": "${MEMORY_REQUEST}"
+                  },
+                  "limits": {
+                    "cpu": "${CPU_LIMIT}",
+                    "memory": "${MEMORY_LIMIT}"
+                  }
+                },
+                "volumeMounts": [
+                  {
+                    "name": "${BACKUP_VOLUME_NAME}",
+                    "mountPath": "${BACKUP_DIR}"
+                  },
+                  {
+                    "name": "${VERIFICATION_VOLUME_NAME}",
+                    "mountPath": "${VERIFICATION_VOLUME_MOUNT_PATH}"
+                  },
+                  {
+                    "name": "${NAME}-config-volume",
+                    "mountPath": "${CONFIG_MOUNT_PATH}${CONFIG_FILE_NAME}",
+                    "subPath": "${CONFIG_FILE_NAME}"
+                  }
+                ]
+              }
+            ]
+          }
+        }
+      }
+    }
+  ],
+  "parameters": [
+    {
+      "name": "NAME",
+      "displayName": "Name",
+      "description": "The name assigned to all of the resources.  Use 'backup-{database name}' depending on your database provider",
+      "required": true,
+      "value": "backup-mongodb"
+    },
+    {
+      "name": "SOURCE_IMAGE_NAME",
+      "displayName": "Source Image Name",
+      "description": "The name of the image to use for this resource.  Use 'backup-{database name}' depending on your database provider",
+      "required": true,
+      "value": "backup-mongodb"
+    },
+    {
+      "name": "IMAGE_NAMESPACE",
+      "displayName": "Image Namespace",
+      "description": "The namespace of the OpenShift project containing the imagestream for the application.",
+      "required": true,
+      "value": "backup-container"
```

Comment:
Should this be esm?

Reply:
will switch this, was defined by params file